spec/lib/capistrano/dsl_spec.rb

```diff
@@ -77,7 +77,7 @@ class DummyDSL
         dsl.invoke("some_task")
         expect do
           dsl.invoke("some_task")
-        end.to output(/.*Capistrano tasks may only be invoked once.*/).to_stderr
+        end.to output(/If you really meant to run this task again, use invoke!/).to_stderr
```

Comment:
Can you add a corresponding spec for `invoke!` that checks that this message *isn't* printed?